### PR TITLE
fril.jp

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/116925

<details><summary>Screenshot:</summary>

Снимок экрана 2022-04-29 в 19 49 16

Снимок экрана 2022-04-29 в 19 49 40

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -25,6 +25,7 @@
 m.facebook.com##div[data-type="vscroller"] > div[style="margin-top:5px; height:167px; z-index:0; width:360px;"][data-mcomponent="MContainer"][data-tti-phase]
 storm.mg#%#//scriptlet("set-cookie", "app_download_banner", "true")
 storm.mg###floating_bottom_ad
+fril.jp##section[class^="mainVisual"]
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/mobile-app.txt:28]: Isn't `##.mainVisual` enough?

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -44,6 +44,8 @@ oppai-paipai.com##.movie-player-block
 ||js.gazo.space/tagjpa.php?
 cndata.jpg4.pw###myinvidad
 cndata.jpg4.pw###thbss > div > a[onclick][title="CLOSE"]
+fril.jp###mkdiv_header_pitari
+fril.jp##.campaignBanners
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:48]: Not an ad, just a PR of fundraising for Ukrine by the same entity. I don't know if worth block even as annoyance @Alex-302 @zloyden

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (Alex-302) [reply JapaneseFilter/sections/specific.txt:48]: Current contant may be changed.

I see ads here:

<details>
<summary>Details</summary>

image


</details>

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:48]: They're still self-promo.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**